tests/test.py

```diff
@@ -15,7 +17,19 @@ def read_lines():
     tree = ast.parse(s)
     file_tokens = tokenize.tokenize(io.BytesIO(s.encode("utf-8")).readline)
     plugin = Plugin(tree, read_lines, file_tokens)
-    return {f"{line}:{col + 1} {msg}" for line, col, msg, _ in plugin.run()}
+    file_tokens = tokenize.tokenize(io.BytesIO(s.encode("utf-8")).readline)
+    plugin2 = Plugin_2(tree, read_lines, file_tokens)
```

Comment:
- Please improve the naming of the plugins :)
 - You could have two test files that each only test one plugin. Give the naming of the test files some thought =)